src/main/java/duke/util/SerializeUtil.java

```diff
@@ -0,0 +1,57 @@
+package duke.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+/**
+ * A utility class designed to serialize/deserialize obects
+ * that are serializable.
+ */
+public class SerializeUtil {
+
+    /**
+     * Serialize an object in to byte array.
+     *
+     * @param object Obect to be serialized
+     * @return serialized object in the form of byte array
+     */
+    public static byte[] serialize(Object object) {
```

Comment:
Perhaps you can include in the name what you are serializing to make it clearer?